lib/Service/Appointments/BookingCalendarWriter.php

```diff
@@ -94,7 +94,12 @@ private function secondsToIso8601Duration(int $secs): string {
 	 * @return string
 	 * @throws RuntimeException
 	 */
-	public function write(AppointmentConfig $config, DateTimeImmutable $start, string $displayName, string $email, ?string $description = null) : string {
+	public function write(AppointmentConfig $config,
+						  DateTimeImmutable $start,
+						  string $displayName,
+						  string $email,
+						  ?string $description = null,
+						  ?string $location = null) : string {
```

Comment:
The location is on the $config object (unless you want to refactor that and allow a talk room per booking, not per config?)